xml/System.ComponentModel.DataAnnotations/ValidationContext.xml

```diff
@@ -367,6 +367,7 @@ In .NET Framework 4.7.2 and earlier versions, when using a custom <xref:System.C
    </appSettings>
 </configuration>
 ```
+In .NET 4.8 RTM, this preoperty returns member name without the configuration setting.
```

Comment:
```suggestion

In .NET Framework 4.8 RTM, this property returns member name without the configuration setting.
```

Reply:
Not so sure customers know what RTM means and how they will know which version they're using

Reply:
Could I change it to .NET 4.8 version before the October update?

Reply:
Could I change it to .NET 4.8 version before the October update?  Or do you have a better suggestion?